libqtile/backend/x11/core.py

```diff
@@ -103,7 +107,7 @@ def __init__(self, display_name: str = None) -> None:
         )
 
         self._wmname = "qtile"
-        self._supporting_wm_check_window = self.conn.create_window(-1, -1, 1, 1)
+        self._supporting_wm_check_window = xcbq.Window(self.conn, self.conn.create_wid(-1, -1, 1, 1))
```

Comment:
Should these be `self.Window`? Or am I missing the point of the class level class properties added to core?

Reply:
The idea is that the core needs to provide the Window class defined by that backend to the rest of libqtile in a general way. Class attributes seem like a convenient way to do that. A direct import e.g. using `xcbq.Window` is specific to the x11 backend so cannot be used outside of the x11 backend to get the Window. In the case of the core's definition itself (like this line) it is equivalent though. However, I think the final form of the backends might not need to provide a Window instance at all, but time will tell.